composer/trainer/trainer_hparams.py

```diff
@@ -265,8 +269,160 @@ def validate(self):
                 "val_dataset and evaluators shouldn't both be specified. Only one can be passed in to the trainer.")
 
     def initialize_object(self) -> Trainer:
-        from composer.trainer.trainer import Trainer
-        return Trainer.create_from_hparams(hparams=self)
+        from composer.trainer import Trainer
```

Comment:
can the import be at the top?

will probably need to be `from composer.trainer.trainer import Trainer` to avoid a circular import error

Reply:
Fair enough. It's "standard practice" in other hparams that we import the object at the start of `initialize_object`, but it's just a given that the `Trainer` is gonna need to be imported sooner or later anyways.

Reply:
For context, we did this because the object itself imported the hparams class. But since we're not doing this anymore the import can be at the top